partner_search_alias/views/res_partner_views.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_res_partner_filter" model="ir.ui.view">
+        <field name="name">res.partner.select</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_res_partner_filter" />
+        <field name="arch" type="xml">
+            <field name="name" position="attributes">
+                <!-- add search_alias to name search -->
+                <attribute
+                    name="filter_domain"
+                >['|', '|', '|', '|', '|', ('display_name', 'ilike', self), ('ref', '=', self), ('email', 'ilike', self), ('vat', 'ilike', self), ('company_registry', 'ilike', self), ('search_alias', 'ilike', self)]</attribute>
+            </field>
+        </field>
+    </record>
```

Comment:
We might want to consider the approach of extending the get_view() method instead, in a similar manner as https://github.com/oca/sale-workflow/blob/3625fea/sale_partner_selectable_option/models/sale_order.py#L12-L24. That way we should be able to avoid the risk of conflicts.

Reply:
I still think this approach can lead to unintended behavior if there are some modules that add new domain at the first place of domain lists (eg, `[('new_field', '=', 'value'), '|', '|', '|', '|', ('display_name', 'ilike', self), ('ref', '=', self), ('email', 'ilike', self), ('vat', 'ilike', self), ('company_registry', 'ilike', self)]`)

Reply:
Wouldn't that be cleared by using `expression.OR()`?

Reply:
Should we consider there will be only OR concatenations for domain of name search? If so, yes.
In my opinion, I consider there is rare chance of using `AND` concatenation for name search domain.

Reply:
@AungKoKoLin1997 I can't imagine a case where `AND` is suitable for the name search. Besides, I don't see the downside of using `expression.OR()` compared to the domain override if that works.